File: Modul 5/Bahasa C/PRAK504-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h> 

int reverse(int bilangan) {
    int reverse_bilangan = 0;
    
    while (bilangan > 0) {
        int digit = bilangan % 10;
        reverse_bilangan = reverse_bilangan * 10 + digit;
        bilangan /= 10;
    }
    return reverse_bilangan;
}
int main() {
    int A, B;
    printf("Input\n");
    scanf("%d %d", &A, &B);
    printf("\nOutput\n");
    A = reverse(A);
    B = reverse(B);
    int C = A + B;
    printf("%d", reverse(C)); 
}

File: Modul 5/Bahasa Python/PRAK504-2310817120001-DinaIzzatiElfadheya.py
def reverse(bilangan):
    reversed_bilangan = 0

    while bilangan > 0:
        digit = bilangan % 10
        reversed_bilangan = reversed_bilangan * 10 + digit
        bilangan //= 10

    return reversed_bilangan
print("Input")
A, B = map(int, input().split())

A = reverse(A)
B = reverse(B)
C = A + B
print("\nOutput")
print(reverse(C))
